lib/eex/test/eex_test.exs

```diff
@@ -175,7 +175,7 @@ defmodule EExTest do
     end
 
     test "when end expression has a modifier" do
-      assert_raise EEx.SyntaxError, ~s[nofile:1: unexpected beginning of EEx tag "<%=" on end of expression "<%= end %>", please remove "=" accordingly], fn ->
+      assert ExUnit.CaptureIO.capture_io :stderr, fn ->
```

Comment:
I initially had it as:

```elixir
      assert ExUnit.CaptureIO.capture_io(:stderr, fn ->
        EEx.compile_string "foo <%= if true do %>true<% else %>false<%= end %>"
      end) =~ ~s[beginning of EEx tag \"<%=\" on end of expression \"<%= end %>\", please remove \"=\" accordingly
```

but noticed that the test above (with warning also) does not check the message. Is asserting on message preferred here and there?